.github/workflows/ci_release.yml

```diff
@@ -62,10 +62,28 @@ jobs:
     needs: changes
     runs-on: ubuntu-latest
     if: ${{ (needs.changes.outputs.updates == 'true' && github.event_name == 'schedule') || github.event_name == 'workflow_dispatch' }}
+    outputs:
+      new_version: ${{ steps.version.outputs.new-version }}
```

Comment:
have you checked whether dashed property works with dot accessor ? in javascript that would not work and you would need to do sth like `steps.version.outputs["new-version"]` so I'm just concerned and you're also assigning it to a variable with a underscore which also raises my concern :)

Reply:
yup, this was all tested on the docker test repo. All these PRs are not just copying the same code to all the repos. 

the `-` vs `_` doesn't matter. That is just mis matched conversations between actions. 

I tried to stay consistent in the actions themselves with how the external actions are set up. But then that leads to the inevitable mismatch for things like this. Ultimately there is no way around it, there will be mismatches somewhere. 

yml doesn't care if the variable has a `-` or a `_`